src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -67,6 +68,11 @@ internal void SetErrors(IEnumerable<IDocumentError> errors)
         /// </summary>
         public ReadOnlySymbolTable Symbols { get; set; }
 
+        /// <summary>
+        /// Culture info passed to this binding. May be null. 
+        /// </summary>
+        internal CultureInfo CultureInfo { get; set; }

```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

discussed offline - we can start internal and make it public later.  We'll likely need more than just CultureInfo (may need TimeZoneInfo and other things from config)
There may be several different patterns here.

Reply:
Shouldn't we put that in the Symbols instead?

Reply:
Culture info can be added to Symbols. If symbols doesn't contain a culture info service, this one will be taken.

Reply:
Possibly in a future change, but this PR is scoped to just fixing a customer facing bug.  
keeping it internal allows us to make that migration.